lib/java/com/google/android/material/appbar/CollapsingToolbarLayout.java

```diff
@@ -814,6 +820,38 @@ public boolean isTitleEnabled() {
     return collapsingTitleEnabled;
   }
 
+
+  /**
+   * Set ellipsizing on the title text.
+   *
+   * @param ellipsize type of ellipsis behavior
+   * @attr ref R.styleable#CollapsingToolbarLayout_titleTextEllipsize
+   */
+  public void setTitleEllipsize(@NonNull TruncateAt ellipsize) {
+    collapsingTextHelper.setTitleTextEllipsize(ellipsize);
+  }
+
+  private void setTitleEllipsize(int ellipsize) {
+    TruncateAt titleTextEllipsize = TruncateAt.END;
```

Comment:
Let's use default: case to return default value instead.

Reply:
Done